packages/gatsby-source-graphql/README.md

```diff
@@ -150,6 +150,49 @@ module.exports = {
 }
 ```
 
+## Composing Apollo Links for production network setup
+
+The plugin provides `createLink` option that you can use to enhance your network stack:
+add error handling, retries, timeouts, etc. Check out [this article](https://medium.com/@joanvila/productionizing-apollo-links-4cdc11d278eb)
```

Comment:
For a11y, it's best practice to use specific link text (instead of "this article" or "here"), so that people using screen readers know where the link is about to take them.

https://www.washington.edu/accessibility/links/
```suggestion
add error handling, retries, timeouts, etc. Check out this [Medium article, Productionizing Apollo Links](https://medium.com/@joanvila/productionizing-apollo-links-4cdc11d278eb),
```